### PR TITLE
adding 6.1 & 6.2 test step to the build and upload

### PR DESCRIPTION
**Pull request by moishce**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://github.com/demisto/etc/issues/36316

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -601,7 +601,15 @@ references:
         context: bucket_upload_env
         requires:
           - Run Unit Testing And Lint
-          - Run Validations
+#          - Run Validations
```
Comment (ShahafBenYakir) [inline .circleci/config.yml:604]: Can we revert?

**Review by ShahafBenYakir — APPROVED** [review]
Looks ok on my end @guyfreund @avidan-H can you take a quick look and approve?

**Comment by avidan-H** [comment]
Shouldn't we remove server version 5.0.0 in the build then?

**Review by guyfreund — APPROVED** [review]
From my side, just need to revert what @ShahafBenYakir said

**Pull request merged**